.github/workflows/prow_cron_pull_request_merge.yml

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 The Serverless Workflow Specification Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: "Merge on lgtm label"
+on:
+  schedule:
+  - cron: "30 * * * *"
```

Comment:
is there a way to do this without cron? a job that will respond to comments?

Reply:
prow-github-action didn't provide a way to invoke this beyond cron job. we nee move to https://github.com/marketplace/actions/merge-pull-requests-automerge-action for this purpose (but i haven't test it).